### PR TITLE
Adding Proxy How-to guide

### LINKED ISSUE #71: Document full story about how to use elemental behind proxy
I would like a How-to Guides section in the documentation to tell how to configure elemental to be used behind a proxy.

In `sidebars.js` :
```
{
      "type": "category",
      "collapsible": true,
      "collapsed": false,
      "label": "How-to Guides",
      "items": [
        "elemental-behind-proxy"
      ]
    },
```

### PR DESCRIPTION
**Pull request by juadk**
Fix #71

### CODE REVIEW

**Review by davidcassany — APPROVED** [review]
LGTM :+1:

File: docs/elemental_behind_proxy.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+![Add proxy settings in Machine Registration](images/proxy-settings-machine-registration-ui.png)
+</TabItem>
+</Tabs>
+
+
+
+## Elemental-register
+
+[Elemental-register](https://elemental.docs.rancher.com/architecture/#elemental-register-client) is the first communication endpoint between the new host and Rancher Manager, this is the first place where proxy settings could be needed.
```
Comment (nunix) [inline docs/elemental_behind_proxy.md:46]:
```suggestion
[Elemental-register](https://elemental.docs.rancher.com/architecture/#elemental-register-client) is the first communication endpoint between the new host and Rancher Manager, this is the first place where proxy settings need to be set.
```

File: docs/elemental_behind_proxy.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+:::warning warning
+At the time of writing, that's only possible to configure proxy settings for the ISO in CLI, nothing is implemented in the UI yet ( and we do not know if it will be, ask for it!)
```
Comment (nunix) [inline docs/elemental_behind_proxy.md:49]:
```suggestion
At the time of writing, it's only possible to configure proxy settings for the ISO with the CLI. The proxy settings aren't implemented in the UI.
```

File: docs/elemental_behind_proxy.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+:::
+
+The process happens when you boot your Elemental ISO for the first time, in order to configure proxy for it, you have to include a cloud-init definition in the ISO.
+To do that, you have to create a [SeedImage](https://elemental.docs.rancher.com/seedimage-reference/) definition.
```
Comment (nunix) [inline docs/elemental_behind_proxy.md:53]:
```suggestion
The process happens when you boot your Elemental ISO for the first time, in order to configure the proxy settings you have to include a `cloud-init` definition in the ISO.
To do that, you have to create a [SeedImage](https://elemental.docs.rancher.com/seedimage-reference/) definition.
```

File: docs/elemental_behind_proxy.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+<CodeBlock language="yaml" title="seedimage.yaml" showLineNumbers>{SeedimageProxy}</CodeBlock>
+
+Apply the YAML with kubectl and then, print your SeedImage definition to get the URL to download it:
```
Comment (nunix) [inline docs/elemental_behind_proxy.md:57]:
```suggestion
Apply the YAML with `kubectl` and then, print your SeedImage definition to get the URL to download it:
```

File: docs/elemental_behind_proxy.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+```bash showLineNumbers
+kubectl get seedimage <seed_image_name> -n <namespace> -o yaml
```
Comment (nunix) [inline docs/elemental_behind_proxy.md:60]: As the description before the codeblock mentions an apply, can you add it here too?

File: docs/elemental_behind_proxy.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+```
+
+Boot the ISO and you should see your new system appears in [Machine inventory](https://elemental.docs.rancher.com/architecture#machineinventory).
+
+## Create Elemental cluster
+For that step you can also use either UI or CLI.
```
Comment (nunix) [inline docs/elemental_behind_proxy.md:66]:
```suggestion

For this step, you can use either the UI or CLI.
```

**Review by nunix — COMMENTED** [review]
Overall change to active voice and few sentences rewrites.
Added a `caution` admonition to highlight the Rancher manager proxy settings not being covered.

**Review by nunix — APPROVED** [review]

**Pull request merged**